private-templates-service/client/src/components/NavBar/NavBar.tsx

```diff
@@ -0,0 +1,127 @@
+import React, { ReactElement } from "react";
+import {
+  Collapse,
+  Container,
+  Navbar,
+  NavbarToggler,
+  NavbarBrand,
+  Nav,
+  NavItem,
+  NavLink,
+  UncontrolledDropdown,
+  DropdownToggle,
+  DropdownMenu,
+  DropdownItem
+} from "reactstrap";
+
+// Components
+import { UserType } from '../../App';
+
+// Styles
+import "@fortawesome/fontawesome-free/css/all.css";
+import { AvatarIcon, DefaultAvatarIcon } from './styled';
+
+function UserAvatar(props: { user: UserType }): ReactElement {
+  // If a user avatar is available, return an img tag with the pic
+  if (props.user.avatar) {
+    return (
+      <AvatarIcon
+        src={props.user.avatar}
+        alt="user"
+        className="rounded-circle align-self-center mr-2"
+        style={{ width: "32px" }}
```

Comment:
Maybe I don't understand how your styling library works (I'll read up about it), but these styles should be in their own file somewhere

Reply:
this was my mistake, it can be removed and styling would still be applied.